doc/developer-notes.md

```diff
@@ -171,3 +171,36 @@ Threads
 - BitcoinMiner : Generates bitcoins (if wallet is enabled).
 
 - Shutdown : Does an orderly shutdown of everything.
+
+Ignoring IDE/editor files
+--------------------------
+
+In closed-source environments in which everyone uses the same IDE it is common
+to add temporary files it produces to the project-wide `.gitignore` file.
```

Comment:
I think the point is that ignore files should always be specific to the project, and not to the developer's environment.